app/controllers/application_controller.rb

```diff
@@ -56,4 +56,38 @@ def store_location
     flash[:error] = exception.message
     redirect_to root_url
   end
+
+  private
+
+  def search_tags
+    index = 0
+    country_tags = Country.uniq.select([:id, :name]).map{ |x|
```

Comment:
hound wants you to replace the `{}` with `do ... end`

I personally don't mind either way